### PR TITLE
BasicEntity: map GET parameters to DELETE method

### PR DESCRIPTION
**Pull request by martenb**
AFAIK DELETE method does not have body, so we can map entity to GET parameters.

### CODE REVIEW

**Comment by f3l1x** [comment]
Could you please add a test?

File: tests/cases/Mapping/RequestEntityMapping.phpt [hunk 1]
```diff
@@ -79,3 +81,27 @@ test(function (): void {
...
+	$request = new ApiRequest(Psr7ServerRequestFactory::fromSuperGlobal());
+	$entity = new NotEmptyEntity();
+
+	$queryRequest = $request
+		->withQueryParams(['foo' => 1]);
+
+	$bodyRequest = $request
+		->withBody(Utils::streamFor(json_encode(['foo' => 1])));
+
+	foreach (['GET', 'DELETE'] as $method) {
```
Comment (f3l1x) [inline tests/cases/Mapping/RequestEntityMapping.phpt:96]: Can you use constants from Endpoint::METHOD_?

**Review by f3l1x — CHANGES_REQUESTED** [review]

File: tests/cases/Mapping/RequestEntityMapping.phpt (hunk 1 above)
Comment (martenb) [reply tests/cases/Mapping/RequestEntityMapping.phpt]: Sure

**Comment by f3l1x** [comment]
👍

**Pull request merged**